dev.md

```diff
@@ -8,6 +8,29 @@ Opportunities are available for everyone, all the way from students looking to
 learn to career scientists looking to fund dedicated open source time. Below
 are our major developer programs with more information on how to get involved.
 
+## SciML Small Grants Program
+
+The SciML small grants program is a program used to fund directed development
+towards high-impact issues through community donations. The small grant projects
+are decided by the SciML Steering Council and candidates can choose to take on
+projects from the project list. This is similar to "bounty programs" seen in
+other open source environments, though it is driven by SciML in a manner that
```

Comment:
What calibre of developer (from first-PR undergrad to you 😛) would you expect could solve these issues?  Giving this kind of "guideline" might decrease the intimidation factor, especially for newcomers.

Reply:
added